### PR TITLE
Fix readme links

### LINKED ISSUE #1049: Broken links in README
Since reorganizing the docs, some of the links in the README are broken. Make a pass through the README and ensure no dead links.

### PR DESCRIPTION
**Pull request by caleb-johnson**
Fixes #1049 

We refactored docs but did not update all the links in README. This PR cleans up the hyperlinks in the README

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -71,14 +71,12 @@ For user convenience, this section assumes that users will deploy the infrastruc
...
-   
-   Save [this example python code](https://qiskit-extensions.github.io/quantum-serverless/quickstart/index.html#id8).  
+1. Write your first example Qiskit Pattern.
+   In the JupyterLab, create a new file, `pattern.py`, in the `work` directory. You can include any arbitrary Python code in your program, or you can use the
+   [example Python file in this tutorial](https://github.com/caleb-johnson/quantum-serverless/blob/main/docs/getting_started/basic/01_running_program.ipynb).
 
-   Now, you are ready to run the first program.
 1. Run the program
-   In the JupyterLab, create a new notebook in the same directory as your program, and execute [this python code](https://qiskit-extensions.github.io/quantum-serverless/quickstart/index.html#id9).
+   In the JupyterLab, create a new notebook in the same directory as your program, and execute [the tutorial code](https://github.com/caleb-johnson/quantum-serverless/blob/main/docs/getting_started/basic/01_running_program.ipynb).
```
Comment (IceKhan13) [inline README.md:79]: `https://github.com/caleb-johnson/q` ? 😄

Comment (caleb-johnson) [reply README.md]: lol I'm trying to drive up traffic to my fork ;)

Nice catch

**Review by IceKhan13 — APPROVED** [review]
Thank you @caleb-johnson !

**Comment by caleb-johnson** [comment]
> Thank you @caleb-johnson !

I should have caught this in the initial PR :)

This is fine to merge since it isn't a feature?

**Comment by IceKhan13** [comment]
yes, let's merge it!

**Pull request merged**